dds_cli/directory.py

```diff
@@ -29,16 +29,23 @@
 class DDSDirectory:
     """Data Delivery System directory class."""
 
-    def __init__(self, path=pathlib.Path, add_file_dir: bool = True):
+    def __init__(
+        self,
+        path=pathlib.Path,
+        add_file_dir: bool = True,
```

Comment:
Just wondering what the purpose of `add_file_dir` is, since I don't see it being used anywhere in the code

Reply:
It's used on line below.

Reply:
I meant that it doesn't seem to be set anywhere and passed to the class init, so it always defaults to True. Or am I missing something?